core/trino-spi/src/main/java/io/trino/spi/security/AccessDeniedException.java

```diff
@@ -315,7 +315,7 @@ public static void denyUpdateTableColumns(String tableName, Set<String> updatedC
 
     public static void denyUpdateTableColumns(String tableName, Set<String> updatedColumnNames, String extraInfo)
```

Comment:
Don't you need to fix any test? Can you please add some if there are no tests.  See `io.trino.security.TestAccessControl`